application/src/test/java/org/opentripplanner/apis/gtfs/GraphQLIntegrationTest.java

```diff
@@ -226,6 +242,24 @@ public Set<Route> getRoutesForStop(StopLocation stop) {
     };
     routes.forEach(transitService::addRoutes);
 
+    // Crate a calendar (needed for testing cancelled trips)
```

Comment:
My PR in #6245 also does something similar and we will need to coordinate.

Reply:
I discussed this with @leonardehrenfried and didn't end up changing the tests in this scope as implementing what I need would have been slightly more difficult with mocking but we can revisit the topic how we should test RealTime updates in the scope of your pr.